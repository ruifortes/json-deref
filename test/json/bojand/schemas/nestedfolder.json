{
  "title": "nested folder test",
  "definitions": {
    "f1": {
      "$ref": "folder1/withinf1.json"
    },
    "someNestedProp" : {
      "$ref": "folder1/more/more.json#/definitions/plusOne"
    }
  },
  "properties": {
    "f1": {
      "$ref": "#/definitions/f1"
    },
    "someNestedProp": {
      "$ref": "#/definitions/someNestedProp"
    }
  }
}