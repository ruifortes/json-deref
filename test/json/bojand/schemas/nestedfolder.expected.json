{
  "title": "nested folder test",
  "definitions": {
    "f1": {
      "description": "Just a basic schema within folder1.",
      "title": "Folder1 Object",
      "type": "object",
      "properties": {
        "something": {
          "description": "some property",
          "readOnly": true,
          "type": "number"
        },
        "someMore": {
          "type": "number"
        }
      }
    },
    "someNestedProp": {
      "type": "string"
    }
  },
  "properties": {
    "f1": {
      "description": "Just a basic schema within folder1.",
      "title": "Folder1 Object",
      "type": "object",
      "properties": {
        "something": {
          "description": "some property",
          "readOnly": true,
          "type": "number"
        },
        "someMore": {
          "type": "number"
        }
      }
    },
    "someNestedProp": {
      "type": "string"
    }
  }
}